core/internal/cltest/simulated_backend.go

```diff
@@ -54,6 +54,11 @@ func NewApplicationWithConfigAndKeyOnSimulatedBlockchain(
 	chainId := backend.Blockchain().Config().ChainID
 	cfg.Overrides.DefaultChainID = chainId
 
+	// Only set P2PEnabled override to false if it wasn't set by calling test
+	if !cfg.Overrides.P2PEnabled.Valid {
+		cfg.Overrides.P2PEnabled = null.BoolFrom(false)
+	}
```

Comment:
Isn't it off by default in the standard config?

Reply:
I think it defaults to true in tests because of https://github.com/smartcontractkit/chainlink/blob/b3f9450e55264f473eadf90218a27a77c46ce74d/core/internal/testutils/configtest/p2p_v1_config.go#L16

Reply:
maybe its simpler for that to not return a port, and then the tests that need to explicitly enable P2P can just set a P2PListenPort and we don't need this override?

Reply:
github is really laggy today

Reply:
yeah thats what I mean, one less override to deal with. Is that a big change?

Reply:
spoke offline - prod config also uses a random port, so better to keep it aligned. Not worth it to force nops to set a port given v1 ocr is deprecated.